portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -12,21 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// TODO: Modify servlet to access multiple messages from servlet.
```

Comment:
nit: usually you want to add the TODO below the class comment. So for this case, under line 28.

Reply:
But do not include in the JavaDoc, because that is public.

Reply:
Fixed